config/rbac/role.yaml

```diff
@@ -111,7 +111,6 @@ rules:
   - networking.k8s.io
   resources:
   - networkpolicies
-  - ingresses
```

Comment:
@vaibhavjainwiz the changes in this file was done automatically just by running `make test`.

Is this expected?

Reply:
This PR should not change manifests I think